gemfiles/rails_60_webpacker.gemfile

```diff
@@ -0,0 +1,11 @@
+source "https://rubygems.org"
+
+eval_gemfile(File.expand_path(File.join("..", "Gemfile.common"), __dir__))
+
+gem "rails", "~> 6.0.0"
+gem "activerecord-jdbcsqlite3-adapter", "~> 60.0.rc1", platform: :jruby
+gem "sqlite3", "~> 1.4", platform: :mri
+
+gem "webpacker", "~> 4.x"
```

Comment:
I think we can use `4.0` here? Or do you need a specific `webpacker` pre-release for this?